src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -548,15 +548,17 @@
                 }
             },
             "diffReport": {
-                "button": "Show",
-                "tooltip": "Shows the detailed git-diff between the template and solution repositories.",
+                "button": "Review Changes",
+                "tooltip": "Shows the detailed git-diff between the template and solution repository.",
```

Comment:
`tooltip` is marked here as well since I think that it should be "repository" instead of "repositories"